merch/buy/complete.php

```diff
@@ -1,6 +1,6 @@
 <?php
 require('../../template/top.php');
-head('Merch Ordered', true);

```

Comment:
This change shouldn't be needed, since it's just passing in defaults, right?

Reply:
The default description is `null` which puts the default description. Putting an empty string sets the OG description to be empty.